features/kiesrecht.feature

```diff
@@ -1,106 +1,179 @@
-  | einddatum uitsluiting vorig jaar   | A                     | 20210000              |                         |           |
-  | einddatum uitsluiting dit jaar     | A                     | 20220000              | true                    | 20220000  |
-  | einddatum uitsluiting onbekend     | A                     | 00000000              | true                    | 00000000  |
+			| omschrijving                             | uitsluiting kiesrecht | einddatum uitsluiting | uitgeslotenVanKiesrecht | einddatum    | einddatumtype  |
+			| einddatum uitsluiting in toekomst        | A                     | volgend jaar          | true                    | volgend jaar | JaarDatum      |
+			| einddatum uitsluiting morgen             | A                     | morgen                | true                    | morgen       | Datum          |
+			| einddatum uitsluiting deze maand         | A                     | deze maand            | true                    | deze maand   | JaarMaandDatum |
+			| einddatum uitsluiting dit jaar           | A                     | dit jaar              | true                    | dit jaar     | JaarDatum      |
+			| einddatum uitsluiting onbekend           | A                     | 00000000              | true                    | 00000000     | DatumOnbekend  |
+
+		Abstract Scenario: kiesrecht <omschrijving>
+		#   Functioneel zou dit moeten zijn :
+		#   Gegeven de persoon met burgerservicenummer '000000140' heeft de volgende 'kiesrecht' gegevens
+		#   De kiesrechtgegevens zijn opgenomen in de lo3_pl tabel. Het toevoegen van kiesrecht gegevens zou in de automation code een update vereisen.
+		#   Die update is nog niet geïmplementeerd, dus worden de kiesrecht met de insert van de inschrijving meegegeven.
+				Gegeven de persoon met burgerservicenummer '000000140' heeft de volgende 'inschrijving' gegevens
+				| aanduiding uitgesloten kiesrecht (38.10) | einddatum uitsluiting kiesrecht (38.20) |
+				| <uitsluiting kiesrecht>                  | <einddatum uitsluiting>                 |
+				Als personen wordt gezocht met de volgende parameters
+				| naam                | waarde                                   |
+				| type                | RaadpleegMetBurgerservicenummer          |
+				| burgerservicenummer | 000000140                                |
+				| fields              | uitsluitingKiesrecht                     |
+				Dan heeft de response een persoon zonder gegevens
+
+			Voorbeelden:
+				| omschrijving                             | uitsluiting kiesrecht | einddatum uitsluiting |
+				| einddatum uitsluiting in verleden        | A                     | vorig jaar            |
+				| einddatum uitsluiting vandaag            | A                     | vandaag               |
+				| einddatum uitsluiting vorige maand       | A                     | vorige maand          |
+				| einddatum uitsluiting vorig jaar         | A                     | vorig jaar            |
```

Comment:
Deze is hetzelfde als die op regel 176. Wat is de meerwaarde hiervan?

Reply:
Regel 176 aangepast naar gisteren. Dit is nu een volledige datum waarmee vergeleken wordt terwijl vorige maand en vorig jaar gedeeltelijk gevulde datums zijn. In deze vorm heeft deze volgens mij nu wel meerwaarde.